jdbc/pom.xml

```diff
@@ -328,7 +335,14 @@
                       <exclude>dnsjava:*</exclude>
                       <exclude>io.airlift:*</exclude>
                       <exclude>io.dropwizard.metrics:*</exclude>
-                      <exclude>io.netty:*</exclude>
+                      <exclude>io.netty:netty-handler</exclude>
```

Comment:
@ayushtkn @abstractdog Could you help to check if this change would cause regression for HIVE-24138? 

I just included `io.netty:netty-common` to fix the `NoClassDefFoundError: io/netty/util/NetUtil when using hive jdbc standalone jar`.

Thanks.